### ISSUE TITLE
Multiple should_fail per file

### ISSUE THREAD

**Issue by zuiderkwast** [josefs/Gradualizer#129]
It would be nice to have multiple functions that should fail in the same test module.

A simple solution for this would be to check that the test returns as many errors as the number of exported functions. (Helper functions are often added and these should not fail.) This can be done after #112.

**Comment by zuiderkwast** [comment]
Exporting the test functions also gets rid of compilation warnings for the test files.

**Comment by Zalastax** [comment]
How do we do this while also testing the error reporting? Passing in `return_errors` will lower the test coverage. Should `should_fail_test` run both with and without `return_errors`?
